applications/imaging_ai_segmentator/README.md

```diff
@@ -1,198 +1,157 @@
+The input for this application is a folder of DICOM image files from a CT series. For testing, CT scan images can be downloaded from [The Cancer Imaging Archive](https://nbia.cancerimagingarchive.net/nbia-search/), subject to [Data Usage Policies and Restrictions](https://www.cancerimagingarchive.net/data-usage-policies-and-restrictions/).
 
-**_Note_**:
-Please download, or otherwise make available, DICOM files of a CT Abdomen series and save them in a folder, preferably named `data/imaging_ai_segmentator/dicom` under the project root, as this folder name is used in the examples in the following steps. Manual download scripts are shown in [`Run the Application in Dev Environment`](#run-the-application-in-dev-environment)
+One such data set, a CT Abdomen series described as `ABD/PANC_3.0_B31f`, was used in testing the application. Other DICOM CT Abdomen series can be downloaded from TCIA as test inputs.
 
 ### Data Citation
 
-National Cancer Institute Clinical Proteomic Tumor Analysis Consortium (CPTAC). (2018). The Clinical Proteomic Tumor Analysis Consortium Cutaneous Melanoma Collection (CPTAC-CM) (Version 11) [Dataset]. The Cancer Imaging Archive. https://doi.org/10.7937/K9/TCIA.2018.ODU24GZE
+National Cancer Institute Clinical Proteomic Tumor Analysis Consortium (CPTAC). (2018). The Clinical Proteomic Tumor Analysis Consortium Cutaneous Melanoma Collection (CPTAC-CM) (Version 11) [Dataset]. The Cancer Imaging Archive. <https://doi.org/10.7937/K9/TCIA.2018.ODU24GZE>
 
+## Model
 
-## Run Instructions
+This application uses the [MONAI whole-body segmentation model](https://github.com/Project-MONAI/model-zoo/tree/dev/models/wholeBody_ct_segmentation), which can segment 104 body parts from CT scans.
 
-There are a number of ways to build and run this application, as well as packaging this application as a Holoscan Application Package. The following sections describe each in detail.
+## Build and Run Instructions
 
 ### Quick Start Using Holohub Container
 
-This is the simplest and fastest way to see the application in action running as a container. The input DICOM files must first be downloaded and saved in the folder `$PWD/data/imaging_ai_segmentator/dicom`, whereas the PyTorch model is automatically downloaded when container image is built.
-
-Use the following to build and run the application:
+This is the simplest and fastest way to run the application:
 
 ```bash
-mkdir -p output
-rm -rf output/*
-./dev_container build_and_run imaging_ai_segmentator --container_args "-v $PWD/output:/var/holoscan/output -v $PWD/data/imaging_ai_segmentator/dicom:/var/holoscan/input"
+rm -rf output
+./dev_container build_and_run imaging_ai_segmentator
 ```
 
-Once the command completes, please check the output folder for the results, e.g.
-```
+**_Note:_** It takes quite a few minutes when this command is run the first time.
+
+The output will be available in the `${HOLOSCAN_OUTPUT_PATH}` directory:
```

Comment:
Because the instruction to set env vars in the original document has been removed in this PR, the env var would not have been set as the user follows the doc to the tee.